spring-test/src/main/java/org/springframework/test/web/servlet/htmlunit/webdriver/MockMvcHtmlUnitDriverBuilder.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2002-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.springframework.test.web.servlet.htmlunit.webdriver;
+
+import com.gargoylesoftware.htmlunit.BrowserVersion;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.htmlunit.MockMvcWebConnectionBuilderSupport;
+import org.springframework.test.web.servlet.setup.MockMvcConfigurer;
+import org.springframework.web.context.WebApplicationContext;
+
+/**
+ * Convenience class for building an HtmlUnitDriver that will delegate to MockMvc and
+ * optionally delegate to an actual connection for specific requests.
+ *
+ * By default localhost will delegate to MockMvc and any other URL will delegate
```

Comment:
Thanks. I've updated the documentation in the feature branch now (and _rewritten history_ again :wink:).